packages/meditrak-server/src/database/models/Option.js

```diff
@@ -4,14 +4,21 @@
  **/
 
 import { DatabaseModel, DatabaseType, TYPES } from '@tupaia/database';
-import { hasContent } from '@tupaia/utils';
 
 class OptionType extends DatabaseType {
   static databaseType = TYPES.OPTION;
 
   static fieldValidators = new Map()
     .set('value', [
-      hasContent,
+       async (value) => {
+        const checkIsEmpty = value => value === undefined || value === null || value.length === 0;
```

Comment:
Also you should be seeing a couple of lint errors in this function if your linter is working correctly so it would be good to look into that when you get a chance. 

eg.
<img width="748" alt="Screen Shot 2020-09-21 at 11 55 43 AM" src="https://user-images.githubusercontent.com/12807916/93725221-76694a00-fc01-11ea-9cc5-076e19d7e07b.png">